src/Atomizer/Renderer.php

```diff
@@ -255,9 +256,10 @@ private function columnOptions(AbstractColumn $column): array
             $options['size'] = $column->getSize();
         }
 
-        if ($column->getAbstractType() === 'decimal') {
-            $options['scale'] = $column->getScale();
-            $options['precision'] = $column->getPrecision();
+        foreach ($column->getAttributes() as $attribute => $value) {
```

Comment:
Can't we just render all the attributes? Are there wrong attributes?

Reply:
I thought it was not safe. Made a rendering of all attributes.